README.md

```diff
@@ -80,6 +80,8 @@ To add a new feature :
 - Implement your method in the child class (e.g [TwitterClient.java](https://github.com/redouane59/twittered/blob/master/src/main/java/com/github/redouane59/twitter/TwitterClient.java))
 - Don't forget to add your unit tests in `src/test/java/com/github/redouane59/twitter/unit`
 
+Code style is also available in `GoogleStyle.xml` file.
+
```

Comment:
## IDE settings

I would let IntelliJ handle this by doing:

![image](https://user-images.githubusercontent.com/6195718/94440085-7fc65980-01a1-11eb-9516-5de6f939f8d2.png)

Then clicking on :gear: > `Copy to project...`.
You should then have two files in `.idea/codeStyles`.
Unignore them from your `.gitignore` and commit them.

## Other IDE support

I would then add in the `README.md` a line on where to find the style (`.idea/codeStyles/<STYLE_NAME>.xml`) in case other developers were using another IDE.

Reply:
`Copy to project...` just gave me an xml file in `/.idea`. I guess I should create on my side the `/codeStyles` folder and add the style file. Doing it :)